angular/core/components/dismiss_explanation_modal/dismiss_explanation_modal.coffee

```diff
@@ -0,0 +1,9 @@
+angular.module('loomioApp').factory 'DismissExplanationModal', ->
+  templateUrl: 'generated/components/dismiss_explanation_modal/dismiss_explanation_modal.html'
+  controller: ($scope, thread, Records, FlashService) ->
+    $scope.thread = thread
```

Comment:
Looks like you're not using $scope.thread anywhere. I reckon we just want to say $scope.thread.dismiss() on line 7, though.